src/transformers/utils/quantization_config.py

```diff
@@ -418,3 +428,53 @@ def post_init(self):
                     f"""dataset needs to be either a list of string or a value in
                     ['wikitext2','c4','c4-new','ptb','ptb-new'], but we found {self.dataset}"""
                 )
+        if self.bits == 4:
+            if self.disable_exllama is not None:
+                logger.warning(
+                    "Using `disable_exllama` is deprecated and will be removed in version 4.37. Use `use_exllama` instead and specify the version with `exllama_version`."
+                    "The value of `use_exllama` will be overwritten by `disable_exllama` passed in `GPTQConfig` or stored in your config file."
+                )
+                self.use_exllama = not self.disable_exllama
+
+            if self.use_exllama:
+                if self.exllama_version == ExllamaVersion.ONE:
+                    logger.warning(
```

Comment:
This doesn't need to be at a warning level

```suggestion
                    logger.info(
```